### PR TITLE
Fix width button

### PR DESCRIPTION
**Pull request by sanmiawi**
Allow setting of button width.

Screenshot 2022-01-24 at 13 00 51
Screenshot 2022-01-24 at 13 01 58

### CODE REVIEW

**Review by emandilara — APPROVED** [review]

File: pyrene/src/components/Button/Button.tsx [hunk 1]
```diff
@@ -76,6 +81,7 @@ const Button: React.FC<ButtonProps> = ({
       }
       onClick={onClick}
       disabled={disabled}
+      style={{ width: width }}
```
Comment (maximelovino) [inline pyrene/src/components/Button/Button.tsx:84]: this doesn't have a unit then, it's an error no? shouldn't it be 
```
style={{ width: `${width}px`}}
```
instead?

**Review by maximelovino — CHANGES_REQUESTED** [review]

File: pyrene/src/components/Button/Button.tsx (hunk 1 above)
Comment (maximelovino) [reply pyrene/src/components/Button/Button.tsx:84]: Also, in this case you need to a check if `width` is undefined before as well

Comment (sanmiawi) [reply pyrene/src/components/Button/Button.tsx:84]: If it is a `number` as specified by the type, React automatically adds the px, and if width is `undefined` then width is is not set.

Comment (maximelovino) [reply pyrene/src/components/Button/Button.tsx:84]: Ok, that works for me

**Review by maximelovino — APPROVED** [review]

**Pull request merged**